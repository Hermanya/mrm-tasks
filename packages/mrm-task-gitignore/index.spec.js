'use strict';

jest.mock('fs');
jest.mock('mrm-core/src/util/log', () => ({
	added: jest.fn(),
}));

const getConfigGetter = require('../../mrm').getConfigGetter;
const vol = require('memfs').vol;
const task = require('./index');

afterEach(() => vol.reset());

it('should add .gitignore', () => {
	vol.fromJSON();

	task(getConfigGetter({}));

	expect(vol.toJSON()).toMatchSnapshot();
});
