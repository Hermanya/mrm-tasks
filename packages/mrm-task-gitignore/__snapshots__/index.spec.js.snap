// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should add .gitignore 1`] = `
Object {
  "/.gitignore": "node_modules/
.DS_Store
Thumbs.db
.idea/
.vscode/
*.sublime-project
*.sublime-workspace
*.log
.eslintcache",
}
`;
